OpenTrons/opentrons-api
# feat(app): add uis for csv file

# Overview
Update protocol card, pinned protocol and parameters screen.
The codes are from my USB PR #15551

close AUTH-460 and AUTH-542

https://github.com/Opentrons/opentrons/assets/474225/dda63a14-5afe-448c-bcd3-296d0f8be442





# Test Plan
1. send a protocol that requires a csv file to Flex
2. go to all protocols
  check the sent protocol's protocol card has yellow background and chip
3. tap the protocol card
4. tap start set up
  check there is csv requires button in Parameters screen



# Changelog
- update protocol card, pinned protocol card, carousel, and their tests
- update protocol setup parameters screen


# Review requests



# Risk assessment
low
